src/solids4FoamModels/solidModels/solidFoam/solidFoam.H

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------------------------*\
+License
+    This file is part of solids4foam.
+
+    solids4foam is free software: you can redistribute it and/or modify it
+    under the terms of the GNU General Public License as published by the
+    Free Software Foundation, either version 3 of the License, or (at your
+    option) any later version.
+
+    solids4foam is distributed in the hope that it will be useful, but
+    WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with solids4foam.  If not, see <http://www.gnu.org/licenses/>.
+
+Class
+    linGeomSolid
+
+Description
+    Mathematical model where linear geometry is assumed i.e. small strains and
+    small rotations are assumed, where the increment of displacement is the
+    primary unknown.
+
+    The stress is calculated by the run-time selectable mechanical law.
+
+Author
+    Philip Cardiff, UCD.  All rights reserved.
```

Comment:
Of course, when you copy the code from solidFoam, make sure to include the original authors.